go.mod

```diff
@@ -4,5 +4,6 @@ go 1.12
 
 require (
 	github.com/davecgh/go-spew v1.1.1
+	gopkg.in/laverya/yaml.v3 v3.0.0-beta1
```

Comment:
why include v3 from your fork. what differences it has from upstream?

Reply:
Two major ones - the first is that it has tags
The second is the inclusion of https://github.com/go-yaml/yaml/pull/455 - "Add a function to set the desired line length of an Encoder" which would resolve https://github.com/kubernetes-sigs/kustomize/issues/947

Reply:
The full diff is here: https://github.com/go-yaml/yaml/compare/v3...laverya:v3.0.0-beta1
7 additional commits